config/jvm_client_options

```diff
@@ -19,6 +19,9 @@
 -Xms256m
 -Xmx512m
 
+# Fix LOG4J2-2052
+-DAsyncLogger.ThreadNameStrategy=UNCACHED
```

Comment:
move to here？
https://github.com/apache/seatunnel/blob/ca94e3beb0f4b5325e781839c2d3abb7659578da/seatunnel-core/seatunnel-starter/src/main/bin/seatunnel-cluster.sh#L86

Reply:
done.